Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -308,6 +308,8 @@ wxMenuBar* CFrame::CreateMenu()
 	platformMenu->Check(IDM_LIST_GC, SConfig::GetInstance().m_ListGC);
 	platformMenu->AppendCheckItem(IDM_LIST_WAD, _("Show Wad"));
 	platformMenu->Check(IDM_LIST_WAD, SConfig::GetInstance().m_ListWad);
+	platformMenu->AppendCheckItem(IDM_LIST_ELFDOL, _("Show ELFs/DOLs"));
```

Comment:
Inconsistent with "Show Wad". Either change this to "Show ELF/DOL" or change "Show Wad" to "Show WADs".